src/libraries/System.Private.CoreLib/src/System/Globalization/CompareInfo.WebAssembly.cs

```diff
@@ -119,6 +125,34 @@ private unsafe int JsIndexOfCore(ReadOnlySpan<char> source, ReadOnlySpan<char> t
             return idx;
         }
 
+        private char[] emptyCharsToRemove = {
+            '\u200d', '\u200b', '\u200c', '\uFEFF', '\u200E', '\u200F',
+            '\u2060', '\u2063', '\u2061', '\u2062', '\u2064', '\u180E',
+            '\u202A', '\u202B', '\u202D', '\u202E', '\u2066', '\u2067',
+            '\u2068', '\u2069', '\u202C'
+        };
+
+        private ReadOnlySpan<char> RemoveEmptyChars(ReadOnlySpan<char> source, CompareOptions options)
```

Comment:
it could also lower-case it, rename it to `SanitizeForInvariantHash` or something like that ?

Reply:
I like the same, changing